### PR TITLE
BGPKiller: Add version 0.9.6.0

### PR DESCRIPTION
**Pull request by issaclin32**
**BGPKiller** ([homepage](http://bgpkiller.weebly.com)) is an ad-blocker for [Avira Antivirus](https://www.avira.com/en/free-antivirus).

Notes:
* I followed [the manifest of SmartTaskBar](https://github.com/lukesampson/scoop-extras/blob/master/bucket/smarttaskbar.json) to provide "Add to startup" for BGPKiller.

* **license**: The license terms are written in the "license" section on the homepage.

* **HTTPS**: The website(bgpkiller.weebly.com) does not support HTTPS.

* **Download URI**: [Archived page from 2015](https://web.archive.org/web/20150303204158/http://bgpkiller.weebly.com:80/download.html) shows that the download URI (uploads/4/1/2/2/41220059/...) has not been changed over time.

### CODE REVIEW

File: bucket/bgpkiller.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            "    'Write-Host \"BGPKiller has been added to the startup programs.\"'",
+            "    'Write-Host \"Run bgpkiller-remove-startup to remove from startup.\"'",
+            ")",
+            "Set-Content \"$dir\\bgpkiller-add-startup.ps1\" -Value ($cont -join \"`r`n\") -Encoding Ascii",
+            "Set-Content \"$dir\\bgpkiller-remove-startup.ps1\" -Value \"Remove-Item '$startup\\BGPKiller.lnk' -ErrorAction SilentlyContinue -Force\" -Encoding Ascii"
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "\"$dir\\bgpkiller-remove-startup.ps1\"",
```
Comment (Ash258) [inline bucket/bgpkiller.json]:
```suggestion
            "& \"$dir\\bgpkiller-remove-startup.ps1\"",
```

File: bucket/bgpkiller.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            "bgpkiller.exe",
+            "BGPKiller"
+        ]
+    ],
+    "checkver": {
+        "url": "http://bgpkiller.weebly.com/download.html",
+        "regex": "bgpkiller_setup_v([\\d.]+).exe"
+    },
+    "autoupdate": {
+        "url": "http://bgpkiller.weebly.com/uploads/4/1/2/2/41220059/bgpkiller_setup_v$version.exe"
```
Comment (Ash258) [inline bucket/bgpkiller.json]: Is this URl static across updates?

Comment (issaclin32) [reply bucket/bgpkiller.json]: Yes. 
[Archived page from 2016](https://web.archive.org/web/20160305121146/http://bgpkiller.weebly.com/download.html) shows that the download URI (uploads/4/1/2/2/41220059/...) has not been changed over time.

File: bucket/bgpkiller.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            "    'Write-Host \"Run bgpkiller-remove-startup to remove from startup.\"'",
+            ")",
+            "Set-Content \"$dir\\bgpkiller-add-startup.ps1\" -Value ($cont -join \"`r`n\") -Encoding Ascii",
+            "Set-Content \"$dir\\bgpkiller-remove-startup.ps1\" -Value \"Remove-Item '$startup\\BGPKiller.lnk' -ErrorAction SilentlyContinue -Force\" -Encoding Ascii"
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "& \"$dir\\bgpkiller-remove-startup.ps1\"",
+            "taskkill /F /IM bgpkiller.exe /FI \"status eq running\""
```
Comment (Ash258) [inline bucket/bgpkiller.json]:
```suggestion
            "taskkill /F /IM bgpkiller.exe /FI 'status eq running'"
```

**Pull request merged**